docs/core/introduction.md

```diff
@@ -179,231 +77,172 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
 
 ## Runtime libraries
 
-.NET has an expansive standard set of class libraries, known as [runtime libraries](../standard/glossary.md#runtime), [framework libraries](../standard/glossary.md#framework-libraries), or the [base class library (BCL)](../standard/glossary.md#bcl). These libraries provide implementations for many general-purpose and workload-specific types and utility functionality.
+.NET has an expansive standard set of class libraries. These libraries provide implementations for many general-purpose and workload-specific types and utility functionality.
 
 Here are some examples of types defined in the .NET runtime libraries:
 
 * Every .NET type derives from the <xref:System.Object?displayProperty=fullName> type.
-* Primitive types, such as <xref:System.Boolean?displayProperty=nameWithType> and <xref:System.Int32?displayProperty=nameWithType>.
+* Primitive value types, such as <xref:System.Boolean?displayProperty=nameWithType> and <xref:System.Int32?displayProperty=nameWithType>.
 * Collections, such as <xref:System.Collections.Generic.List%601?displayProperty=nameWithType> and <xref:System.Collections.Generic.Dictionary%602?displayProperty=nameWithType>.
 * Data types, such as <xref:System.Data.DataSet?displayProperty=nameWithType> and <xref:System.Data.DataTable?displayProperty=nameWithType>.
 * Network utility types, such as <xref:System.Net.Http.HttpClient?displayProperty=nameWithType>.
 * [File and stream I/O](../standard/io/index.md) utility types, such as <xref:System.IO.FileStream?displayProperty=nameWithType> and <xref:System.IO.TextWriter?displayProperty=nameWithType>.
 * [Serialization](../standard/serialization/index.md) utility types, such as <xref:System.Text.Json.JsonSerializer?displayProperty=nameWithType> and <xref:System.Xml.Serialization.XmlSerializer?displayProperty=nameWithType>.
 * High-performance types, such as <xref:System.Span%601?displayProperty=nameWithType>, <xref:System.Numerics.Vector?displayProperty=nameWithType>, and [Pipelines](../standard/io/pipelines.md).
 
-For more information, see the [Runtime libraries overview](../standard/runtime-libraries-overview.md). The source code for the libraries is in [the GitHub dotnet/runtime repository](https://github.com/dotnet/runtime/tree/main/src/libraries).
+For more information, see the [Runtime libraries overview](../standard/runtime-libraries-overview.md).
 
-### Extensions to the runtime libraries
+## NuGet Package Manager
 
-Libraries for some commonly used application functionality aren't included in the runtime libraries but are made available in NuGet packages, such as the following:
+[NuGet](/nuget/what-is-nuget) is the package manager for .NET. It enables developers to share compiled binaries with each other. [NuGet.org](https://www.nuget.org/) offers many [popular packages](https://www.nuget.org/stats/packages) from the community.
+
+The following table lists NuGet packages from the .NET team:
```

Comment:
Good call. Are they the most important? This is the existing table. I didn't try to improve it. Should we just nuke it?

Reply:
Either nuke it, or change it to be a table of most popular nuget packages (based on nuget.org download counts).

Reply:
Nuking seems fine; I don't think the ones listed are the most important nor the most representative. It would entirely depend on the kind of app you're building IMHO.

We could point to the .NET team NuGet user names but there are also a lot of packages that we don't really expect developers to reference (e.g. all the runtime and ref packs). Sadly, those are downloaded a lot hence they tend to rank highly and thus paint a very "deep in the weeds" or "gory implementation details".

*If* we want to include some NuGet packages my recommendation would be to pick packages that are app model neutral, generally referenced as a package (as opposed to being auto referenced as part of a shared framework), and not legacy (provided for backwards compatibility and/or porting from .NET Framework).

Good candidates would be:

* EF Core (`Microsoft.EntityFrameworkCore`)
* SQL Client (`Microsoft.Data.SqlClient`)
* IoT (`System.Device.Gpio`)

Reply:
I nuked it. We can come with something different if we want later.